# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

Header header
geometry_msgs/Pose2D pose
float32 speed
float32 radius
